flow/util/genRuleFile.py

```diff
@@ -358,6 +343,18 @@ def comma_separated_list(value):
         default=None,
         help="Reference metadata file.",
     )
+    parser.add_argument(
+        "--rules",
+        type=str,
+        default=None,
+        help="Rules input file.",
+    )
+    parser.add_argument(
+        "--new-rules",
+        type=str,
+        default=None,
+        help="Rules input file.",
```

Comment:
This would be the output, right?

Reply:
yes: I have tried to be consistent with the "old" vs "new" terminology in the code. However, I was thinking that perhaps later, I'd prefer to have an input/output terminology which is less surprising than the "new" "old" terminology in this file.